Development environment/Docker.md

```diff
@@ -1,56 +1,155 @@
-In order to install a WordPress development environment on [Docker](https://www.docker.com/), you'll need to install Docker and [docker-compose](https://docs.docker.com/compose/).
+In order to install a WordPress development environment on [Docker](https://www.docker.com/), you'll need to install Docker and use [docker-compose](https://docs.docker.com/compose/).
+
+Once you have those installed, in your desired folder add `docker-compose.yml` file
 
-Once you have those installed, install your WordPress in the folder of your choice, and add `docker-compose.yml` file to the root of your WordPress installation folder.
 
 ```yml
-version: '3'
+version: '3.3'
+
 services:
-  wordpress:
+
+  app:
+    image: wordpress:5.2-fpm
+    container_name: wp-docker-app
     depends_on:
       - db
-    image: wordpress:latest
-    ports:
-      - 8080:80
-      - 443:443
+    restart: always
     volumes:
-      - ./data:/data # Required if importing an existing database
-      - ./:/var/www/html   # Theme development
+      - ./bin/php.ini:/usr/local/etc/php/conf.d/local.ini
+      - .:/var/www/html
     environment:
-      WORDPRESS_DB_NAME: db_name # pull this from env variables
-      WORDPRESS_DB_HOST: db:3306
-      WORDPRESS_DB_USER: root
-      WORDPRESS_DB_PASSWORD: root
+      WORDPRESS_DB_HOST: wp-docker-db
+      WORDPRESS_DB_NAME: wordpress-handbook-app
+      WORDPRESS_DB_USER: wp
+      WORDPRESS_DB_PASSWORD: wp
+    expose:
+      - "80"
+
   db:
     image: mysql:5.7
+    container_name: wp-docker-db
+    restart: always
     volumes:
-      - data:/var/lib/mysql
+      - db_data:/var/lib/mysql
     environment:
-      MYSQL_ROOT_PASSWORD: root
-      MYSQL_DATABASE: db_name
-      MYSQL_USER: root
-      MYSQL_PASSWORD: root
+      MYSQL_RANDOM_ROOT_PASSWORD: 1
+      MYSQL_DATABASE: wordpress-handbook-app
+      MYSQL_USER: wp
+      MYSQL_PASSWORD: wp
+    ports:
+      - "33066:3306"
+
   phpmyadmin:
     image: phpmyadmin/phpmyadmin
+    container_name: wp-docker-phpmyadmin
+    external_links:
+      - db
+    ports:
+      - 8181:80
     environment:
-      MYSQL_ROOT_PASSWORD: root
+      MYSQL_RANDOM_ROOT_PASSWORD: 1
+      MYSQL_USERNAME: root
+
+  nginx:
+    image: nginx
+    container_name: wp-docker-nginx
+    restart: always
+    volumes:
+      - ./:/var/www/html
+    depends_on:
+      - app
+    volumes:
+      - ./bin/nginx.conf:/etc/nginx/conf.d/default.conf
+      - .:/var/www/html
+      - ./logs:/var/log/nginx
     ports:
-      - 3306:80
+      - 8010:80
+
 volumes:
-  data: {}
+  db_data:
+
 ```
 
-After adding that file and changing the database name, run
+Besides adding `docker-compose.yml` file, create `bin` folder with two files in it: `nginx.conf` and `php.ini` for your nginx and php settings. The `nginx.conf` looks like this
```

Comment:
Besides adding the `docker-compose.yml` file, create a `bin` folder with two files in it—`nginx.conf` and `php.ini`—for your nginx and php settings. The `nginx.conf` file looks like this